src/meson.build

```diff
@@ -0,0 +1,111 @@
+src_files = files(
```

Comment:
Unless there was a specific reason for this, I generally use a regular array than a `files()` object.

The only difference is `files()` keeps the path if you pass it around other directories.